File: flutter/lib/config/env.dart
class Env {
  static String get backend_http_url =>
      const String.fromEnvironment('BACKEND_HTTP_URL', defaultValue: 'https://doantotnghiep-working.onrender.com');

  static String get backend_ws_url =>
      const String.fromEnvironment('BACKEND_WS_URL', defaultValue: 'wss://doantotnghiep-working.onrender.com/ws');

  static String get default_device_id =>
      const String.fromEnvironment('DEFAULT_DEVICE_ID', defaultValue: 'child_01');

  static double get default_lat => 21.0285;

  static double get default_lng => 105.8542;

  static int get geofence_radius_m => 100;
}

File: frontend/src/config/env.js
export const env = {
  backendHttpUrl: import.meta.env.VITE_BACKEND_HTTP_URL || "http://localhost:9000",
  backendWsUrl: import.meta.env.VITE_BACKEND_WS_URL || "ws://localhost:9000/ws",
  deviceId: import.meta.env.VITE_DEVICE_ID || "child_01",
  defaultLat: Number(import.meta.env.VITE_DEFAULT_LAT || 21.0285),
  defaultLng: Number(import.meta.env.VITE_DEFAULT_LNG || 105.8542),
  geofenceRadiusM: Number(import.meta.env.VITE_GEOFENCE_RADIUS_M || 100),
};
